setup.cfg

```diff
@@ -1,2 +1,59 @@
 [metadata]
+author = Argo AI
+author_email = argoverse-api@argo.ai
+classifiers = 
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: MacOS
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Artificial Intelligence
 description_file = README.md
+keywords =
+    self-driving-car
+    dataset-tools
+license = MIT
+name = argoverse
+url = https://www.argoverse.org
+version = attr: argoverse.VERSION
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+install_requires =
+    colour
+    descartes
+    imageio
+    h5py
+    hydra-core ==1.1.0
+    lap
+    matplotlib
+    motmetrics ==1.1.3
+    numpy ==1.19
+    omegaconf ==2.1.0
+    opencv-python >=4.1.0.25
+    pandas >=0.23.1
+    pillow
+    imageio
+    pyntcloud >=0.1.0
+    scipy >=1.4.0
+    shapely
+    sklearn
+    tqdm
+    typing_extensions
+
+[options.package_data]
+argoverse = argoverse/visualization/data/**/*
+
+[options.packages.find]
+exclude =
+    integration_tests
+    map_files
+    tests
+
+[flake8]
+enable_extensions = G,argoverse
+ignore = E203,E704,E711,E722,E741,W291,W293,W391,W503,F821,F401,F811,F841,P101,G004,G002,I201,I100,I101
+max_line_length = 120
```

Comment:
`flake8` configuration in `setup.cfg` -- does not support `pyproject.toml` currently.